app/components/chart/stats.js

```diff
@@ -131,15 +132,24 @@ class Stats extends Component {
     const stats = [];
 
     const addStat = statType => {
-      stats.push(getStatDefinition(dataUtil[this.dataFetchMethods[statType]](), statType, {
+      const chartStatOpts = _.get(props, ['chartPrefs', chartType, statType]);
+
+      const stat = getStatDefinition(dataUtil[this.dataFetchMethods[statType]](), statType, {
         bgSource,
         days,
         bgPrefs: {
           bgBounds,
           bgUnits,
         },
         manufacturer,
-      }));
+        ...chartStatOpts,
+      });
+
+      if (statType === 'averageDailyDose' && typeof props.onAverageDailyDoseInputChange === 'function') {
```

Comment:
I feel a slight preference to use `_.isFunction` here, but it's mostly just a preference.

Reply:
I agree.  I'll change this.